### DISCUSSION TITLE
Mania scroll speed tooltip isn't rounded anymore in comma separator languages

### DISCUSSION THREAD

**Discussion by Joppe27** [ppy/osu#23752] [Q&A] [answered]
Result of #23691 but possibly a lower-level issue.

Since the above PR, the scroll speed setting's millisecond tooltip is not rounded anymore in languages that use a comma as a decimal separator. This means the entire `double` is displayed to the user which probably wasn't the intention.

https://github.com/ppy/osu/blob/25d0f0f3da506aa0af19edf58c8fe4222a3f6532/osu.Game/Localisation/RulesetSettingsStrings.cs#L85

I can't immediately see the reason for why this wouldn't work because the numeric formatting is there, which is why I think this might be something culture-bound (which I haven't looked into). 

Additionally, I know of another place with an issue that could be related: the BPM and scroll speed text boxes in the editor also do not work correctly when using a comma as a separator.

Example on release (Dutch): 
![osu_2023-06-05_16-27-33](https://github.com/ppy/osu/assets/56885706/598e7681-c795-4518-96d9-2e1f9b47fd2e)

**Comment by monochrome22** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/23750.

**Reply by Joppe27** [reply]
Oops, I only searched discussions. Thanks!
